js/templates/modals/settings/store.html

```diff
@@ -0,0 +1,119 @@
+                <ul class="unstyled errorList hide js-selectedModsError">
+                  <li><i class="ion-alert-circled"></i><span class="js-errorText"></span> </li>
+                </ul>
+                <div class="contentBox boxProcessingFaded modBox clrBr clrP <% if (ob.originalMods) print('hasMods ') %>js-modListSelected">
+                  <%= ob.spinner() %>
+                  <div class="noModsAdded js-noModsAdded">
+                    <div class="txCtr">
+                      <h5 class="txUnl">
+                        <%= ob.polyT('settings.storeTab.selectedModeratorsEmpty') %>
+                      </h5>
+                      <p class="clrT2">
+                        <%= ob.polyT('settings.storeTab.selectedModeratorsEmptyWarning') %>
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <ul class="unstyled errorList hide js-submitModByIDInputError">
+                <li><i class="ion-alert-circled"></i><span class="js-errorText"></span> </li>
+              </ul>
+              <div class="flex gutterH">
+                <input
+                    type="text"
+                    class="btnHeight clrBr clrP clrSh2 js-submitModByIDInput"
+                    placeholder="<%= ob.polyT('settings.storeTab.moderatorByIDPlaceholder') %>">
+                <%= ob.processingButton({
+                className: 'btn clrP clrBr js-submitModByID ',
+                btnText: ob.polyT('settings.storeTab.moderatoryByIDSubmit')
+                }) %>
+              </div>
+              <div>
+                <div class="contentBox boxProcessingFaded modBox modBoxByID clrBr clrP js-modListByID"><!-- unhide when adding moderator-->
+                  <%= ob.spinner() %>
+                </div>
+              </div>
+              <hr class="clrBr unleaded">
+              <div>
+                <div class="rowLg"><% // just a spacer %></div>
+              </div>
+              <div>
+                <h5><%= ob.polyT('settings.storeTab.availableModerators') %></h5>
+                <div class="contentBox boxProcessingFaded modBox clrBr clrP js-modListAvailable">
+                  <%= ob.spinner() %>
+                  <div class="noModsAdded js-noModsAdded">
+                    <button class="btn clrP clrBr js-browseMods">
+                       <%= ob.polyT('settings.storeTab.browseModerators') %>
+                    </button>
+                  </div>
+                  <button class="btnTxt tx6 clrT2 txU browseMore js-browseMods">
+                    <%= ob.polyT('settings.storeTab.browseMoreModerators') %>
+                  </button>
+                </div>
+              </div>
+            </div><% // end of flex inside col9 %>
+            <% if (ob.errors.storeModerators) print('<div class="row"></div>  ' + ob.formErrorTmpl({ errors: ob.errors.storeModerators })) %>
```

Comment:
No need to change this, but as an FYI you can pass a class (or classes) into the error template in case you want to give it a custom style (like some margins):

```
ob.formErrorTmpl({
  errors: ob.errors.storeModerators,
  class: 'class1 class2',
}))
```

Which would result in this:

```
<ul class="unstyled errorList class1 class2">
  <li><i class="ion-alert-circled"></i>You must provide the salamander.</li>
</ul>
```